cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/UserTasksParameters.java

```diff
@@ -16,10 +16,16 @@
 /**
  * Parameters for {@link com.linkedin.kafka.cruisecontrol.servlet.EndPoint#USER_TASKS}
  *
+ * <ul>
+ *   <li>Note that if "fetch_complete_response" is set to true, one and only one user task id should to be set for "user_task_ids"
```

Comment:
The reason we add this parameter is to provide a safer and more convenient way to retrieve the response of a completed task,  this constraint does comply with that intention.

And I feel to return response of multiple tasks will make the logic here unnecessarily complex , the response maybe hard to read, and bring little practical value.

I assume the common usage will be 
1) a request to `user_tasks` and get full list of tasks
2)for the one user interested, they use `user_tasks?uuid=xxx& fetch_complete_response =true`(instead of using the original URL and manipulate UUID) to get complete response.

Reply:
I don't understand why it is `safer` or `more convenient` to have this limitation. 

For the former, could you clarify why it is not safe to return multiple values -- i.e. what is the safety issue? For the latter, if the user wants to get a single response, can't s/he use a single user task id to do so -- i.e. what changes for the user to cause an inconvenience?

> And I feel to return response of multiple tasks will make the logic here unnecessarily complex , the response maybe hard to read, and bring little practical value.

`Practical value`: user can retrieve multiple completed user tasks without repetitive network latency. Also the user can be the UI or another CC client that processes the returned response before presenting it to the end-users.

While the assumptions about the use-cases might be valid, I still think that we should preserve the current behavior of getting multiple uuids using the relevant parameter.

Reply:
By `safer and more convenient ` I am comparing with the old way of doing this([step 2 described in issue 531](https://github.com/linkedin/cruise-control/issues/531)).

I think it is OK to add the support.